src/main/java/uk/gov/hmcts/ccd/endpoint/ui/QueryEndpoint.java

```diff
@@ -290,6 +269,7 @@ public CaseEventTrigger getEventTriggerForDraft(@PathVariable("uid") String user
         @ApiResponse(code = 200, message = "Displayable case data"),
         @ApiResponse(code = 404, message = "Invalid jurisdiction/case type/case reference or event id")
     })
+    @LogAudit(operationType = OperationType.VIEW_CASE)
```

Comment:
better to introduce a VIEW_CASE_HISTORY operation

Reply:
We will ignore this one.